app/src/main/res/layout/timetable_lesson.xml

```diff
@@ -75,15 +75,16 @@
                     android:layout_height="12dp"
                     android:layout_gravity="center_vertical"
                     android:layout_marginHorizontal="8dp"
-                    android:visibility="@{unread ? View.VISIBLE : View.GONE}"
-                    android:background="@drawable/unread_red_circle" />
+                    android:layout_marginTop="4dp"
+                    android:background="@drawable/unread_red_circle"
+                    android:visibility="@{unread ? View.VISIBLE : View.GONE}" />
 
                 <ImageView
                     android:id="@+id/attendanceIcon"
                     android:layout_width="24dp"
                     android:layout_height="24dp"
                     android:layout_gravity="center_vertical"
-                    android:visibility="gone"
```

Comment:
Keep this visibility attribute (use `tools:visibility` if you just want to show it in the designer) so it's not rendered unnecessarily when it's blank.